emu_base/noise.py

```diff
@@ -0,0 +1,11 @@
+import torch
+
+
+def pick_well_prepared_qubits(eta: float, n: int) -> torch.Tensor:
+    """
+    Randomly pick n booleans such that ℙ(True) = eta
+
+    Returns:
+        A BoolTensor of shape (n,) where each element is True with probability eta.
+    """
+    return torch.rand(n) < eta
```

Comment:
why did you invert the comparison?
`eta` is the probability of having a dark qubit, so this would instead be called `pick_dark_qubits` I think